packages/@aws-cdk/cicd/README.md

```diff
@@ -0,0 +1,31 @@
+## Continuous Integration / Continuous Delivery for CDK Applications
+
+### Getting Started
+In order to add the `PipelineDeployStackAction` to your *CodePipeline*, you need to have a *CodePipeline* artifact that
+contains the result of invoking `cdk synth` on your *CDK App*. You can for example achieve this using a *CodeBuild*
+project.
+
+Then, simply add your stacks to be deployed by your *CodePipeline* like so:
+```ts
+import cicd = require('@aws-cdk/cicd');
+
+// Assuming we already have the following:
+const stackToDeploy: cdk.Stack;
+const synthesizedApp: codepipeline_api.Artifact;
+const deployStage: codepipeline_api.IStage;
+
+// Add actions to deploy the stack in the deploy stage:
+new cicd.PipelineDeployStackAction(this, 'DeployStack', {
```

Comment:
Can I assume there's going to be `PipelineDeployAppAction` as well?

Reply:
Maybe in some future. It would require ability to determine the deployment order of all the stacks and to support cross-environment scenarios. There's also a question about how to model the presence of test stages in between different stages... And how the stage boundaries are defined...